adapta/storage/models/format.py

```diff
@@ -93,6 +99,75 @@ def deserialize(self, data: bytes) -> DataFrame:
         return read_csv(io.BytesIO(data))
 
 
+class PolarsDataFrameParquetSerializationFormat(SerializationFormat[PolarsDataFrame]):
```

Comment:
It is a long name, you could consider just calling it `PolarsParquetSerializationFormat` and similarly for the others

Reply:
I did consider this. However, it will cause naming convention challenges if we'd like serializers for other Polars types, such as polars.Series. I thus opted for the more verbose name.

Reply:
Fair enough